JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/screens/videoPlayer/components/VideoPlayerControls.kt

```diff
@@ -37,10 +40,16 @@ fun VideoPlayerControls(
     contentCurrentPosition: Long,
```

Comment:
Passing the current position as a Long means that this whole composable will recompose any time it changes (which is very very often). Use a lambda instead, or consider passing the player itself and creating progress state on a component level, so you can read the values in the right phase.

Reply:
Thanks for the suggestion. I took the 2nd option and create progress state on VideoPlayerSeeker